release_images.yml

```diff
@@ -107,4 +107,28 @@ release_images:
       neuron_sdk_version: "sdk1.19.0"
       example: False
       disable_sm_tag: True
-      force_release: False
\ No newline at end of file
+      force_release: False
+  9:
+    framework: "pytorch"
+    version: "1.12.1"
+    custom_type: "ec2"
+    arch_type: "graviton"
+    inference:
+      device_types: ["cpu"]
+      python_versions: ["py38"]
+      os_version: "ubuntu20.04"
+      example: False
+      disable_sm_tag: True
+      force_release: False
+  10:
+    framework: "tensorflow"
+    version: "2.9.1"
+    custom_type: "ec2"
+    arch_type: "graviton"
+    inference:
+      device_types: ["cpu"]
+      python_versions: ["py38"]
+      os_version: "ubuntu20.04"
+      example: False
+      disable_sm_tag: True
```

Comment:
this should be set to false - the name is no longer appropriate (sm tag) - but the "short tags" are applied to these images for ease of canary testing

Reply:
updated